.eslintrc.js

```diff
@@ -19,7 +19,19 @@ module.exports = {
   overrides: [
     {
       files: ["rollup.config.js", "*.mjs"],
-      parserOptions: { sourceType: "module" },
+      parserOptions: {
+        sourceType: "module",
+        ecmaVersion: 2018,
```

Comment:
@Mingun I particularly want your opinion on this.  My logic is that this code only runs in Node, and we only support 10+.  This allows the use of async/await, which cleaned up a bunch of callback stuff in the CLI.

Reply:
changed to `['*']`